lang/lpython/files/0003-Install-libs-into-a-sane-location.patch

```diff
@@ -0,0 +1,24 @@
+From c3b6f50f85de1a62d3322c6c5e2b1b831ebc0337 Mon Sep 17 00:00:00 2001
+From: Sergey Fedorov <vital.had@gmail.com>
+Date: Wed, 10 Jan 2024 13:43:09 +0800
+Subject: [PATCH] Install libs into a sane location
+
+---
+ src/bin/CMakeLists.txt | 4 ++--
+ 1 file changed, 2 insertions(+), 2 deletions(-)
+
+diff --git src/bin/CMakeLists.txt src/bin/CMakeLists.txt
+index 72a1a1a11..7c836e4ae 100644
+--- src/bin/CMakeLists.txt
++++ src/bin/CMakeLists.txt
+@@ -73,8 +73,8 @@ endif()
+ 
+ install(TARGETS lpython
+     RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+-    ARCHIVE DESTINATION share/lpython/lib
+-    LIBRARY DESTINATION share/lpython/lib
++    ARCHIVE DESTINATION lib
++    LIBRARY DESTINATION lib
```

Comment:
Might a subdir of lib make more sense? Share certainly doesn't make sense of course.

Reply:
There are just two libraries being installed. I do not see any point in a subdirectory, to be honest, and then a need to pass a flag every time to use it.